NameLayerMain/src/vg/civcraft/mc/namelayer/command/commands/SetDefaultGroup.java

```diff
@@ -44,8 +44,7 @@ public boolean execute(CommandSender sender, String[] args) {
 			return true;
 		}
 		
-		GroupPermission gPerm = gm.getPermissionforGroup(g);
-		if (!gPerm.isAccessible(pType, PermissionType.BLOCKS)){
+		if (!gm.hasAccess(g.getName(), uuid, PermissionType.getPermission("BLOCKS"))){
```

Comment:
Namelayer natively doesn't know what BLOCKS is, though, does it? That's a citadel-owned permission. 

There's no reason someone can't set a default group regardless of BLOCKS perms or not, I've never understood this decision.